src/tools/cephfs/cephfs-shell

```diff
@@ -922,7 +926,12 @@ class CephFSShell(Cmd):
                 try:
                     cephfs.rmdir(path)
                 except libcephfs.Error as e:
-                    set_exit_code_msg(msg=e)
+                    if e.get_error_code() in errors.keys():
+                        set_exit_code_msg(msg=f"rmdir: failed to remove "
+                                              f"{path.decode('utf-8')}: "
+                                              f"{errors[e.get_error_code()]}")
```

Comment:
A `str` type is being passed to to `set_exit_code_msg()`. This sets the exit code for cephfs-shell to 1 which is `Misc`, ignoring `exit_codes` dict.

Reply:
Done, thanks